package.json

```diff
@@ -22,15 +22,20 @@
   "dependencies": {
     "@likecoin/likecoin-eth-util": "^1.0.1",
     "axios": "^0.18.0",
+    "body-parser": "^1.18.3",
     "bs58": "^4.0.1",
+    "cookie-parser": "^1.4.3",
     "cross-env": "^5.1.6",
+    "crypto": "^1.0.1",
```

Comment:
crypto should be built-in for current nodejs version and should not be pull from npm